modular_skyrat/master_files/code/defines/_ammo_defines.dm

```diff
@@ -0,0 +1,5 @@
+#define AMMO_TYPE_LETHAL		"lethal"
+#define AMMO_TYPE_RUBBER		"rubber"
+#define AMMO_TYPE_HOLLOWPOINT	"hollowpoint"
+#define AMMO_TYPE_IHDF 			"ihdf"
+#define AMMO_TYPE_460 			".460"
```

Comment:
You don't really have to define this here, seems abit redundant, just put round_type = ".460" to save a pointless define.

Reply:
fair